### PR TITLE
[Merged by Bors] - chore: don't need group-theoretic exponent to set up finite dimensional vector spaces

### PR DESCRIPTION
**Pull request by kim-em**

### CODE REVIEW

**Comment by kim-em** [comment]
[provoked by this `lake exe graph` output](https://github.com/user-attachments/files/18058542/o.pdf)

**Comment by jcommelin** [comment]
bors d+

**Comment by kim-em** [comment]
bors merge

File: Mathlib/LinearAlgebra/Dimension/ErdosKaplansky.lean [hunk 1]
```diff
@@ -0,0 +1,144 @@
+/-
+Copyright (c) 2018 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Johannes Hölzl, Sander Dahmen,
+Kim Morrison, Chris Hughes, Anne Baanen, Junyan Xu
+-/
```
Comment (alreadydone) [inline Mathlib/LinearAlgebra/Dimension/ErdosKaplansky.lean:6]: The content of this file was added completely by me in [#9159](https://github.com/leanprover-community/mathlib4/pull/9159/files#diff-cb173017e1157ddc4b9f868be06c18ec2a38f8cf82e856e18d59ed49f97395d8R150)
Is there any automation at all to keep authorship record straight in splitting PRs?

Comment (bryangingechen) [reply Mathlib/LinearAlgebra/Dimension/ErdosKaplansky.lean:6]: There isn't yet, but it doesn't seem out of the question. There are tools for example to preserve git history when splitting files that we might want to start recommending: https://github.com/potherca-bash/git-split-file

**Pull request closed without merging**